packages/react-composites/src/composites/common/ControlBar/CommonCallControlBar.tsx

```diff
@@ -127,6 +127,8 @@ export const CommonCallControlBar = (props: CommonCallControlBarProps & Containe
   /* @conditional-compile-remove(teams-meeting-conference) */
   const [showTeamsMeetingConferenceModal, setShowTeamsMeetingConferenceModal] = useState(false);
 
+  const adapterState = props.callAdapter.getState();
```

Comment:
For anything that uses adapter state, we should be going through a selector

Reply:
Good Catch! thanks!